ios/RNZoomUs.m

```diff
@@ -600,14 +600,161 @@ - (void)onClickShareScreen:(UIViewController *)parentVC {
+
+- (void)onMyVideoStateChange {}
+
+- (void)onSpotlightVideoChange:(BOOL)on {}
+
+- (void)onSinkMeetingPreviewStopped {}
+
+- (void)onSinkMeetingActiveVideoForDeck:(NSUInteger)userID {}
+
+- (void)onSinkMeetingVideoQualityChanged:(MobileRTCNetworkQuality)qality userID:(NSUInteger)userID {}
+
+- (void)onSinkMeetingShowMinimizeMeetingOrBackZoomUI:(MobileRTCMinimizeMeetingState)state {}
+
+
+#pragma mark - MobileRTCAudioServiceDelegate
+
+- (void)onSinkMeetingMyAudioTypeChange {
+  MobileRTCMeetingService *ms = [[MobileRTC sharedRTC] getMeetingService];
+  MobileRTCMeetingUserInfo *userInfo = [ms userInfoByID:[ms myselfUserID]];
+
+  [self sendEventWithName:@"MeetingEvent" params:@{
+    @"event": @"myAudioTypeChanged",
+    @"userRole": [self getUserRole:[userInfo userRole]],
+    @"audioType": @([[userInfo audioStatus] audioType]),
+    @"isTalking": @([[userInfo audioStatus] isTalking]),
+    @"isMutedAudio": @((BOOL)([[userInfo audioStatus] audioType] == 2 ? YES : [[userInfo audioStatus] isMuted])),
+    @"isMutedVideo": @((BOOL)![[userInfo videoStatus] isSending]),
+  }];
+}
+
+- (void)onSinkMeetingAudioStatusChange:(NSUInteger)userID audioStatus:(MobileRTC_AudioStatus)audioStatus {
+  MobileRTCMeetingService *ms = [[MobileRTC sharedRTC] getMeetingService];
+
+  if ([ms isMyself:userID]) {
+    MobileRTCMeetingUserInfo *userInfo = [ms userInfoByID:[ms myselfUserID]];
+
+    [self sendEventWithName:@"MeetingEvent" params:@{
+      @"event": @"myAudioStatusChanged",
+      @"userRole": [self getUserRole:[userInfo userRole]],
+      @"audioType": @([[userInfo audioStatus] audioType]),
+      @"isTalking": @([[userInfo audioStatus] isTalking]),
+      @"isMutedAudio": @((BOOL)([[userInfo audioStatus] audioType] == 2 ? YES : [[userInfo audioStatus] isMuted])),
+      @"isMutedVideo": @((BOOL)![[userInfo videoStatus] isSending]),
+    }];
+  }
+}
+
+// TODO: This one is not getting called
+- (void)onSinkMeetingAudioRequestUnmuteByHost {
+  [self sendEventWithName:@"MeetingEvent" event:@"askUnMuteAudio"];
+}
+
+- (void)onSinkMeetingAudioStatusChange:(NSUInteger)userID {}
+
+- (void)onSinkMeetingAudioTypeChange:(NSUInteger)userID {}
+
+- (void)onAudioOutputChange {}
+
+- (void)onMyAudioStateChange {}
+
+
+#pragma mark - MobileRTCUserServiceDelegate
+
+- (void)onMyHandStateChange {}
+
+- (void)onInMeetingUserUpdated {}
+
+- (void)onSinkMeetingUserRaiseHand:(NSUInteger)userID {}
+
+- (void)onSinkMeetingUserLowerHand:(NSUInteger)userID {}
+
+- (void)onSinkUserNameChanged:(NSUInteger)userID userName:(NSString *_Nonnull)userName {}
+
+- (void)onClaimHostResult:(MobileRTCClaimHostError)error {}
+
+- (void)onMeetingHostChange:(NSUInteger)userId {
+    [self sendEventWithName:@"MeetingEvent" params:@{
+    @"event": @"hostChanged",
+    @"userId": @(userId)
+  }];
+}
+
+- (void)onMeetingCoHostChange:(NSUInteger)userId {
+    [self sendEventWithName:@"MeetingEvent" params:@{
+    @"event": @"coHostChanged",
```

Comment:
1. add `coHostChanged` event to `EVENTS.md` under `  //   IOS ONLY EVENTS`
2. move `myAudioStatusChanged` and `myVideoStatusChanged` out of `//   ANDROID ONLY EVENTS`